docker/images/kumascript2/Dockerfile

```diff
@@ -0,0 +1,45 @@
+FROM node:10.14.0-slim
```

Comment:
Same deal here: I've got a separate kumascript2/ directory because I wanted to be able to have old and new containers at the same time. This would be fixed before deploying.